Kudu.Services.Web/Content/Scripts/ProcessExplorer.js

```diff
@@ -863,6 +870,14 @@ function overrideRightClickMenu() {
                     $("li")[0].click();
                     $("li").blur();
                     break;
+                case "crashdump":
+                    $("#proc-loading").show();
+                    process.enableCrashDump().done(function () {
+                        return processExplorerSetupAsync();
+                    }).fail(function () {
+                        return processExplorerSetupAsync();
```

Comment:
Does the user get any visual indication that the crash dump registration failed?